Marlin/src/gcode/bedlevel/abl/G29.cpp

```diff
@@ -177,6 +180,20 @@ G29_TYPE GcodeSuite::G29() {
     if (DISABLED(PROBE_MANUALLY) && seenQ) G29_RETURN(false);
   #endif
 
+  TERN_(EXTENSIBLE_UI, ExtUI::onMeshLevelingStart());
+
+  #if ENABLED(PREHEAT_BEFORE_LEVELING)
+    {
```

Comment:
I found this out later. This will cause `G29 W` to heat as well. Needs to be moved further down below.